packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -0,0 +1,344 @@
+
+      this.statsResults[mediaType][sendrecvType].availableBandwidth = kilobytes.toFixed(1);
+      this.statsResults[mediaType].bytesSent = kilobytes.toFixed(1);
+
+      this.statsResults[mediaType][sendrecvType].framesEncoded = result.framesEncoded - this.statsResults.internal[mediaType][sendrecvType].framesEncoded;
+      this.statsResults[mediaType][sendrecvType].keyFramesEncoded = result.keyFramesEncoded - this.statsResults.internal[mediaType][sendrecvType].keyFramesEncoded;
+
+      this.statsResults.internal[mediaType].outboundRtpId = result.id;
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].packetsSent) {
+        this.statsResults.internal[mediaType][sendrecvType].packetsSent = result.packetsSent;
+      }
+
+      this.statsResults[mediaType][sendrecvType].packetsSent = result.packetsSent - this.statsResults.internal[mediaType][sendrecvType].packetsSent;
+      this.statsResults.internal[mediaType][sendrecvType].packetsSent = result.packetsSent;
+      this.statsResults[mediaType][sendrecvType].totalPacketsSent = result.packetsSent;
+    }
+  };
+
+
+  inboundrtp = (result, type) => {
+    if (!result) {
+      return;
+    }
+    if (result.type !== 'inbound-rtp') return;
+    const mediaType = type || 'audio';
+    const sendrecvType = 'recv';
+
+    if (!sendrecvType) return;
+
+    if (result.bytesReceived) {
+      let kilobytes = 0;
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived) {
+        this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived = result.bytesReceived;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].pliCount) {
+        this.statsResults.internal[mediaType][sendrecvType].pliCount = result.pliCount;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].packetsLost) {
+        this.statsResults.internal[mediaType][sendrecvType].packetsLost = result.packetsLost;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].packetsReceived) {
+        this.statsResults.internal[mediaType][sendrecvType].packetsReceived = result.packetsReceived;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].lastPacketReceivedTimestamp) {
+        this.statsResults.internal[mediaType][sendrecvType].lastPacketReceivedTimestamp = result.lastPacketReceivedTimestamp;
+      }
+
+      if (!this.statsResults.internal[mediaType][sendrecvType].nackCount) {
+        this.statsResults.internal[mediaType][sendrecvType].nackCount = result.nackCount;
+      }
+
+      const bytes = result.bytesReceived - (this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived);
+
+      this.statsResults.internal[mediaType][sendrecvType].prevBytesReceived = result.bytesReceived;
+
+      kilobytes = bytes / 1024;
+      this.statsResults[mediaType][sendrecvType].availableBandwidth = kilobytes.toFixed(1);
+      this.statsResults[mediaType].bytesReceived = kilobytes.toFixed(1);
```

Comment:
Seems like a fair amount of duplication between inbound and outbound. Was it more readable this way?